lib/console/console.js

```diff
@@ -141,6 +146,30 @@ export default class InkTerminal extends PaneItem {
     return [this.terminal._core.buffer.x, this.terminal._core.buffer.y]
   }
 
+  setOption (key, val) {
+    if (!this.persistentState) {
+      this.persistentState = { opts: {} }
+    }
+    if (!this.persistentState.opts) {
+      this.persistentState.opts = {}
+    }
```

Comment:
very nitpick, but are these condition checks necessary ? I think `this.persistentState` would be always there (but yeah we don't need to be clean freaks here since this check should be very cheap, though)

Reply:
I think the reason why I added those checks is that `persistentState` isn't set if you upgrade ink from a really old version and then try to deserialize the terminal. Or something like that.